test/integration/targets/groupby_filter/runme.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# We don't set -u here, due to pypa/virtualenv#150
+set -e
```

Comment:
Looks like `-x` got dropped with the last changes. On a side-note, if you use `-x`, you don't need to be so verbose with the messaging about the python version below, since the commands are all shown.